flux-cli/src/test/java/com/marklogic/flux/impl/importdata/ImportDelimitedFilesOptionsTest.java

```diff
@@ -47,4 +49,63 @@ void tdeDirectories() {
             "avoid the awkwardness of using a newline as a delimiter. Commas don't work because they are allowed in " +
             "directory paths (though we still use commas to separate collections as a convenience for users).");
     }
+
+    @Test
+    void tdeColumnCustomizations() {
+        ImportDelimitedFilesCommand command = (ImportDelimitedFilesCommand) getCommand(
+            "import-delimited-files",
+            "--path", "anywhere",
+            "--tde-schema", "test-schema",
+            "--tde-view", "test-view",
+            "--tde-column-val", "column1=customVal1",
+            "--tde-column-val", "column2=customVal2",
+            "--tde-column-type", "column1=dateTime",
+            "--tde-column-type", "column2=int",
+            "--tde-column-default", "column1=defaultValue1",
+            "--tde-column-default", "column2=defaultValue2",
+            "--tde-column-invalid-values", "column1=reject",
+            "--tde-column-invalid-values", "column2=ignore",
+            "--tde-column-reindexing", "column1=visible",
+            "--tde-column-reindexing", "column2=hidden",
+            "--tde-column-permissions", "column1=role1,role2",
+            "--tde-column-permissions", "column2=role3",
+            "--tde-column-nullable", "column1",
+            "--tde-column-nullable", "column2"
+        );
+
+        WriteStructuredDocumentParams params = (WriteStructuredDocumentParams) command.getWriteParams();
+        StructType schema = new StructType()
+            .add("column1", DataTypes.StringType)
+            .add("column2", DataTypes.StringType);
+        TdeInputs inputs = params.buildTdeInputs(schema);
+
+        Map<String, String> columnVals = inputs.getColumnVals();
+        assertEquals("customVal1", columnVals.get("column1"));
+        assertEquals("customVal2", columnVals.get("column2"));
```

Comment:
Would it be worth it to verify that the "standard" values for these settings aren't changed by mistake? Or perhaps that's covered by other tests?